src/lib/dnssd/Resolver.h

```diff
@@ -267,6 +267,22 @@ struct DiscoveredNodeData
             ChipLogDetail(Discovery, "\tPort: %u", port);
         }
         ChipLogDetail(Discovery, "\tCommissioning Mode: %u", commissioningMode);
+        if (mrpRetryIntervalIdle.HasValue())
+        {
+            ChipLogDetail(Discovery, "\tMrp Interval idle: %u ms", mrpRetryIntervalIdle.Value().count());
+        }
+        else
+        {
+            ChipLogDetail(Discovery, "\tMrp Interval idle: not present");
```

Comment:
I think we could just skip this line if it's not present.

Reply:
I wanted to be consisted with the output printed by the Python CHIP controller